gmine_l2.nv.json

```diff
@@ -0,0 +1,74 @@
+{
+  "_notes": [
+    "Compiled by Francis De Brabandere.",
+    "Space Station (Williams 1987)",
+    "https://www.ipdb.org/machine.cgi?id=2261",
+    "Williams System 11B"
+  ],
+  "_copyright": "Copyright (C) 2024 by Francis De Brabandere <francisdb@gmail.com>",
+  "_license": "GNU Lesser General Public License v3.0",
+  "_endian": "big",
+  "_roms": [
+    "spstn_l5"
+  ],
+  "_fileformat": 0.1,
+  "_version": 0.1,
+  "game_state": {
+    "credits": {
+      "label": "Credits",
+      "start": "0x44",
+      "length": 1,
+      "encoding": "bcd"
+    },
+    "default_high_score": {
+      "_note": "Should probably be a adjustment, this is the default high score when pressing the highscore reset button (5 in vpinball). Default is 200",
```

Comment:
You probably don't want to try figuring out all adjustments, but you could start an adjustments section and list this as 12.  Check the High Speed JSON for the Adjustments section.

Reply:
for now removing this as I want to first get all high scores done